src/PVRIptvData.cpp

```diff
@@ -334,11 +335,26 @@ bool PVRIptvData::LoadEPG(time_t iStart, time_t iEnd)
     GetNodeValue(pChannelNode, "category", entry.strGenreString);
     GetNodeValue(pChannelNode, "sub-title", entry.strEpisodeName);
 
-    xml_node<> *pCreditsNode = pChannelNode->first_node("credits");
-    if (pCreditsNode != NULL) {
-        GetNodeValue(pCreditsNode, "actor", entry.strCast);
-        GetNodeValue(pCreditsNode, "director", entry.strDirector);
-        GetNodeValue(pCreditsNode, "writer", entry.strWriter);
+    xml_node<>* pCreditsNode = pChannelNode->first_node("credits");
+    if (pCreditsNode != NULL) 
+    {
+      xml_node<>* pActorNode = nullptr;
+      std::string cast;
+      for(pActorNode = pCreditsNode->first_node("actor"); pActorNode; pActorNode = pActorNode->next_sibling("actor"))
+      {
+        if (pActorNode)
+        {
+          std::string tempString = pActorNode->value();
+          tempString = std::regex_replace(tempString, std::regex(" ?/ ?"), ", "); //replace / with , if there are multiple actors in one line
```

Comment:
maybe that space is not important docs says you should use you should use just EPG_STRING_TOKEN_SEPARATOR 
https://github.com/xbmc/xbmc/blob/master/xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_epg_types.h#L91

Reply:
Possibly not, although it probably won’t matter either way.

Reply:
Let’s see, the code may need other tweaks.

Reply:
for replace `/` -> `,` not necessary, because I added this joint separator;
when builded I check it how kodi display it;
because movie/show info dialog director and writer join with `/` not `,`

Reply:
Ok, then simplifies the code then. Let me push a change and you can check it.

Reply:
Ok, want to test what is there now?

Reply:
If you are happy with it I can just add the version and  adddon xml files.